L1Trigger/L1THGCal/python/hgcalBackEndLayer2Producer_cfi.py

```diff
@@ -2,31 +2,108 @@
+                               dR_multicluster=cms.double(0.01),
+                               minPt_multicluster=cms.double(0.5),  # minimum pt of the multicluster (GeV)
+                               dist_dbscan_multicluster=cms.double(0.),
+                               minN_dbscan_multicluster=cms.uint32(0),
+                               EGIdentification=egamma_identification_drnn_cone.clone(),
+                               )
+
+
+dbscan_C3d_params = cms.PSet(type_multicluster=cms.string('DBSCANC3d'),
+                             dR_multicluster=cms.double(0.),
+                             minPt_multicluster=cms.double(0.5),  # minimum pt of the multicluster (GeV)
+                             dist_dbscan_multicluster=cms.double(0.005),
+                             minN_dbscan_multicluster=cms.uint32(3),
+                             EGIdentification=egamma_identification_drnn_dbscan.clone())
+
+
+histoMax_C3d_params = cms.PSet(type_multicluster=cms.string('HistoMaxC3d'),
+                               dR_multicluster=cms.double(0.03),
+                               dR_multicluster_byLayer_coefficientA=cms.vdouble(),
+                               dR_multicluster_byLayer_coefficientB=cms.vdouble(),
+                               minPt_multicluster=cms.double(0.5),  # minimum pt of the multicluster (GeV)
+                               nBins_R_histo_multicluster=cms.uint32(36),
+                               nBins_Phi_histo_multicluster=cms.uint32(216),
+                               binSumsHisto=binSums,
+                               threshold_histo_multicluster=cms.double(10.),
+                               cluster_association=cms.string("NearestNeighbour"),
+                               EGIdentification=egamma_identification_histomax.clone(),
+                               neighbour_weights=neighbour_weights_1stOrder
+                               )
+# V9 samples have a different defintiion of the dEdx calibrations. To account for it
+# we reascale the thresholds of the clustering seeds
+# (see https://indico.cern.ch/event/806845/contributions/3359859/attachments/1815187/2966402/19-03-20_EGPerf_HGCBE.pdf
+# for more details)
+phase2_hgcalV9.toModify(histoMax_C3d_params,
+                        threshold_histo_multicluster=cms.double(7.5),  # MipT
+                        )
+
+
+histoMaxVariableDR_C3d_params = histoMax_C3d_params.clone()
+histoMaxVariableDR_C3d_params.dR_multicluster = cms.double(0.)
+histoMaxVariableDR_C3d_params.dR_multicluster_byLayer_coefficientA = cms.vdouble(dr_layerbylayer)
+histoMaxVariableDR_C3d_params.dR_multicluster_byLayer_coefficientB = cms.vdouble([0]*53)
+
+
+histoSecondaryMax_C3d_params = histoMax_C3d_params.clone()
+histoSecondaryMax_C3d_params.type_multicluster = cms.string('HistoSecondaryMaxC3d')
+histoSecondaryMax_C3d_params.threshold_histo_multicluster = cms.double(5.)  # FIXME: needs to be adapted in V9 geometry?
```

Comment:
For the moment it's probably better to keep consistent seed thresholds for all algos. Since this is a clone of `histoMax_C3d_params` and the seed threshold is already changed for V9, then this line could just be removed.